### PR TITLE
Project page

### PR DESCRIPTION
**Pull request by abehara2**

### CODE REVIEW

File: api/tests/test_model.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// import React from "react";
```
Comment (yousefa00) [inline api/tests/test_model.js]: You can remove this entirely

File: client/components/Stage.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  CarouselCaption,
+  CarouselItem,
+  FormGroup,
+  Label
+} from "reactstrap";
+import "../public/styles/stage.scss";
+import "../public/styles/style.scss";
+import { Head } from ".";
+export default function(props) {
+  console.log("props.value");
```
Comment (yousefa00) [inline client/components/Stage.jsx]: Logs should be removed

File: client/components/Stage.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import {
+  Button,
+  Card,
+  CardGroup,
+  CardImg,
+  Col,
+  Container,
+  Input,
```
Comment (yousefa00) [inline client/components/Stage.jsx]: We should remove all the imports we're not using

File: client/pages/editProject.jsx [hunk 4]
```diff
@@ -0,0 +1,378 @@
+import {
+  Button,
+  Card,
+  CardGroup,
+  CardImg,
+  Col,
+  Container,
+  Input,
+  Row,
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Same as above

File: client/pages/editProject.jsx [hunk 5]
```diff
@@ -0,0 +1,378 @@
...
+  Carousel,
+  CarouselIndicators,
+  CarouselControl,
+  CarouselCaption,
+  CarouselItem,
+  FormGroup,
+  Label
+} from "reactstrap";
+import {
+  deleteForm,
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: What's deleteForm from?

File: api/tests/test_model.js (hunk 1 above)
Comment (arpanlaha) [inline api/tests/test_model.js]: can we delete this if its unused?

File: client/pages/editProject.jsx [hunk 6]
```diff
@@ -0,0 +1,378 @@
...
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const [title, setTitle] = useState("Loading");
+  const [description, setDescription] = useState("Loading");
+  //Stages
+  let ideationStages = [
+    ["Stage 1", "Description 1"],
+    ["Stage 2", "Description 2"],
+    ["Stage 3", "Description 3"]
+  ];
+  console.log(ideationStages);
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: log

File: client/pages/editProject.jsx [hunk 7]
```diff
@@ -0,0 +1,378 @@
...
+  let addStage = () => {
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Can be const and log

File: client/components/Stage.jsx [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import {
+  Button,
```
Comment (arpanlaha) [inline client/components/Stage.jsx]: can you remove unused imports?

Comment (arpanlaha) [reply client/components/Stage.jsx]: for other changes as well

File: client/components/Stage.jsx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+} from "reactstrap";
+import "../public/styles/stage.scss";
+import "../public/styles/style.scss";
+import { Head } from ".";
+export default function(props) {
+  console.log("props.value");
+  console.log(props);
+  return (
+    <>
+      <Head />
```
Comment (arpanlaha) [inline client/components/Stage.jsx]: components in the `components` folder don't need a `Head`, only those in pages do

File: client/pages/editProject.jsx [hunk 10]
```diff
@@ -0,0 +1,378 @@
...
+  getTitle().then(name => {
+    name = setTitle(name);
+  });
+  //Set Description
+  const getDescription = async () => {
+    const project = await getModelsByID("5e5f20e2f182430b94c8c66d");
+    return project.data[0].description;
+  };
+  getDescription().then(description => {
+    description = setDescription(description);
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Do you need to assign to `description` here or can you just call `setDescription(description)`?

File: client/components/Stage.jsx [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+      <div>
+        <FormGroup style={{ marginTop: "1%", width: "50%" }}>
```
Comment (arpanlaha) [inline client/components/Stage.jsx]: can you move this into a scss file rather than inline?

File: client/components/Stage.jsx (hunk 2 above)
Comment (arpanlaha) [inline client/components/Stage.jsx]: console logs should be removed

File: client/pages/editProject.jsx (hunk 6 above)
Comment (arpanlaha) [inline client/pages/editProject.jsx]: more logs

File: client/pages/editProject.jsx (hunk 7 above)
Comment (arpanlaha) [inline client/pages/editProject.jsx]: should be `const`

File: client/pages/editProject.jsx [hunk 12]
```diff
@@ -0,0 +1,378 @@
...
+  });
+  return (
+    <>
+      <Head title={title} />
+      {/* Icons and buttons and such */}
+      <Row
+        className="header-row"
+        justify="center"
+        align="middle"
+        style={{ marginRight: "15%", marginLeft: "15%", marginBottom: "2.5%" }}
```
Comment (arpanlaha) [inline client/pages/editProject.jsx]: more inline styling

File: client/pages/editProject.jsx [hunk 13]
```diff
@@ -0,0 +1,378 @@
...
+                isOpen={dropdownOpen}
+                toggle={toggle}
+              >
+                <DropdownToggle caret>Choose SDG's</DropdownToggle>
+                <DropdownMenu>
+                  <FormGroup check>
+                    {/* <Label for="exampleSelectMulti">Select Multiple</Label> */}
+                    <Row>
+                      <Label
+                        for="exampleCheck"
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Maybe exampleCheck isn't the most explicit naming scheme to use here

File: client/pages/editProject.jsx [hunk 14]
```diff
@@ -0,0 +1,378 @@
...
+
+              <Dropdown
+                style={{ marginLeft: "10%" }}
+                isOpen={dropdownOpen}
+                toggle={toggle}
+              >
+                <DropdownToggle caret>Choose SDG's</DropdownToggle>
+                <DropdownMenu>
+                  <FormGroup check>
+                    {/* <Label for="exampleSelectMulti">Select Multiple</Label> */}
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Comments can be removed

File: client/pages/editProject.jsx [hunk 15]
```diff
@@ -0,0 +1,378 @@
...
+                        style={{ marginLeft: "1%" }}
+                      />
+                    </Row>
+                    <Row>
+                      <Label
+                        for="exampleCheck"
+                        style={{ marginLeft: "11%" }}
+                        check
+                      >
+                        {" "}
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Is this trying to add a space? If so, we should use the br html tag

File: client/pages/editProject.jsx [hunk 16]
```diff
@@ -0,0 +1,378 @@
...
+          </Button>
+        </Row>
+        <hr
+          style={{ marginRight: "25%", marginLeft: "25%", marginBottom: "0%" }}
+        />
+        <Col
+          style={{ marginLeft: "25%", marginRight: "25%", marginBottom: "1%" }}
+        >
+          {ideationStages.map(value => {
+            console.log(value);
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: log

**Pull request merged**